requirements-test-py26.txt

```diff
@@ -0,0 +1,8 @@
+mock==2.0.0
```

Comment:
@daminijain23  I think this file is not needed anymore.

Reply:
@rbikar removed